index.d.ts

```diff
@@ -0,0 +1,25 @@
+import LRU = require("lru-cache");
```

Comment:
should that be `import LRU from 'lru-cache'`?

Reply:
That's the TypeScript equivalent (it transpires down to) to CommonJS import syntax